src/index.ts

```diff
@@ -0,0 +1,6 @@
+const akamaiTheme: NodeRequire = require('../dist/web/akamai/theme.es6.js');
+const akamaiTokens: NodeRequire = require('../dist/web/akamai/tokens.es6.js');
+const cloudManagerTheme: NodeRequire = require('../dist/web/cloudmanager/theme.es6.js');
+const cloudManagerTokens: NodeRequire = require('../dist/web/cloudmanager/tokens.es6.js');
+
+export { akamaiTheme, akamaiTokens, cloudManagerTheme, cloudManagerTokens };
```

Comment:
Not tied to naming conventions here. I also don't know if we actually want to export individual tokens, but I also can see it being useful for one-offs outside of the theme definition

Reply:
this one is giving me quite a bit of problems so I'll leave it for M3-6533